### ISSUE TITLE
Azurite doesn't check if  the page range specified is valid when perform operations on pageblob

### ISSUE THREAD

**Issue by zzhxiaofeng** [Azure/Azurite#640]
**Error Description:**
In Azure, it will return error "InvalidPageRange" when use Clear Pages, Get Page Ranges and Upload Pages operation with invalid page range on page blob.
But in Azurite, it will perform operations successfully in this case.
Azure error is shown as following:
![Screenshot 2020-11-19 093850](https://user-images.githubusercontent.com/20970631/99609527-0d5c5200-2a4b-11eb-9622-2aebe82f25dd.png)

**To Reproduce**
The demo code is shown as following:
```
import {
 BlobServiceClient
} from "@azure/storage-blob";
import * as assert from "assert"

async function main(){
  const blobServiceClient = await BlobServiceClient.fromConnectionString("<Azurite-https-connectionString>");
  const containerClient=await blobServiceClient.getContainerClient("<container-name>");
  const pageBlobClient = containerClient.getPageBlobClient("<blob-name>");
  await pageBlobClient.createIfNotExists(4*1024);
  try{
    await pageBlobClient.clearPages(5*1024,1024);
  }catch(err){
    console.log(err);
    assert.ok((err as any).response.parsedBody.Code === "InvalidPageRange");
  }
}

main();
```

**Expected Behavior**
When page range specified is invalid, Azurite return the error "InvalidPageRange" in the operation of Clear Pages, Get Page Ranges and Upload Pages on page blob. 

@jongio for notification.

**Comment by blueww** [comment]
@zzhxiaofeng
Thanks for raising the issue! 
Currently Azure Server will fail on the Clear Pages request with invalid range, but Azurite will not return error on it.
We will investigate and evaluate the issue.

We welcome contribution to Azurite.
It would be great if you could contribute the the fix of this issue.

**Comment by blueww** [comment]
This is fixed on Azurite v3.10.0.

**Comment by v-xuto** [comment]
azsdk-azurite
